tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -513,4 +515,203 @@ virtual void TearDown() {
         EXPECT_NEAR(expectedBoxesVert[i].size.height, boxes[i].size.height, c_errorMargin);
     }
 #endif
+}
+
+TEST(CTFont, CreatePathForGlyph) {
+    // Some background on Bezier curves:
+    // A quadratic Bezier curve is specified by 3 points:     a start point, a control point, and an end point
+    // A cubic Bezier curve is instead specified by 4 points: a start point, TWO control points, and an end point
+    // As a generalization, most "older" fonts specify quadratic curves, and "newer" ones can use cubic curves
+    // Times New Roman's curves are all quadratic
+
+    // CGPath has support for both orders of Bezier curve,
+    // but DWrite's GeometrySink only supports cubic Bezier curves, and approximates any quadratic curves in terms of a cubic curve
+    // Eg: Reference platform quadratic curve:  (previous endpoint),              (512, 632),              (480, 632)
```

Comment:
The comments until this point are immensely useful in the context of the implementation code as well, would help to duplicate it there.